docs/index.md

```diff
@@ -1,53 +1,98 @@
-use the [GitHub page](https://github.com/xournalpp/xournalpp) for more
-information.**
-
-Xournal++ is a cross-platform, open source, hand note-taking software with the
-target of flexibility, functionality, and speed. It is a modern rewrite and more
-feature-rich version of the wonderful
-[Xournal](http://sourceforge.net/projects/xournal/) program.
+---
 
-To install Xournal++, please follow to the [installation
-page](installation).
+<link rel = "stylesheet" href = "css/home.css">
+<link rel = "stylesheet" href = "css/responsive.css">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+<meta name="viewport" content="initial-scale=1, maximum-scale=1, width=device-width">
 
-## Some highlighted features
-
-Here are some of what we think are the "killer features" of Xournal++. You can
-find a more complete list of features in the [User Guide](guide).
-
-* Supports a wide variety of devices and user workflows
-    * Pressure-sensitive styluses and digital pen tablets (e.g. Wacom, Huion, XP
-      Pen, etc. tablets)
-    * Plugins support via the [Lua programming language](https://lua.org)
-    * Customizeable toolbar, with multiple configurations, e.g. to optimize toolbar
-      for portrait / landscape
-* Notetaking features
-    * "Paper" backgrounds for notetaking, scratch paper, or whiteboarding
-    * Different pen types (e.g. eraser, highlighter) and stroke styles (e.g.
-      solid, dotted)
-    * Different shapes, such as lines, arrow, circle, rectangle, splines
-    * Shape resizing and rotation
-    * LaTeX support (requires a working LaTeX install)
-    * Multiple "layers" for easier control over overlapping notes and strokes
-    * Sidebar with page previews and PDF Bookmarks
-    * Audio recording and playback alongside with handwritten notes
-    * Rotation and grid snapping
-* File formats
-    * Annotate on top of PDFs
-    * Export to a variety of formats including SVG, PNG, and PDF
-* Other convenience features
-    * Bug reporting, autosave, and auto backup tools
-    * Page template definitions
-    * Multi language support (over 20 languages supported, with more added by the
-      community!)
+<div class = "home">
+	<div class = "Xournal" id = "Xournal">
+		<h1>Take all the notes you want with ease, from solving a physics problem to making travel diaries.</h1>
+		<p>Xournal++ is an open-source and cross-platform note-taking software that is fast, flexible, and functional. A modern rewrite and a more feature-rich version of the wonderful Xournal program.</p>
+		<img src = "img/Xournal++ Headline Example.png">
+		<div class = "download-hyperlink upper-button">
+			<a href = "http://127.0.0.1:8000/installation/">Download Xournal++</a>
```

Comment:
I don't expect this to be done immediately, but these links will eventually need to be relative.

Reply:
Yes I't'll be replaced before merging.